src/pages/splash/splash.less

```diff
@@ -248,6 +248,18 @@
   }
 
   /**
+   * Maker Party Promo
+   */
+  #mp-promo {
+    .promo {
+      .promoMixin(@vikingYellow);
```

Comment:
But if all you're changing is the color, you can just change that value when the design changes, right? Or are there going to be multiple banners up at a time with different colors?